File: beecrowd/1009.cs
// created by mostafizzfuad
using System;
class URI
{
    static void Main(string[] args)
    {

        string sellerName = Console.ReadLine();
        double fixedSalary = Convert.ToDouble(Console.ReadLine());
        double amountOfTotalSell = Convert.ToDouble(Console.ReadLine());

        double bonusSalary = amountOfTotalSell * 0.15;
        double salary = fixedSalary + bonusSalary;

        Console.WriteLine("SALARY = U$ " + salary.ToString("0.00"));

    }
}

File: beecrowd/1009.cpp
// created by mostafizzfuad
#include <bits/stdc++.h>
using namespace std;
int main() {

    string sellerName;
    double fixedSalary, amountOfTotalSell;
    cin >> sellerName >> fixedSalary >> amountOfTotalSell;

    double bonusSalary = amountOfTotalSell * 0.15;
    double salary = fixedSalary + bonusSalary;

    cout << "TOTAL = R$ " << fixed << setprecision(2) << salary << endl;
    
}